source/query/filter.txt

```diff
@@ -40,8 +40,8 @@ Compatibility
 Set Query Filter
 ----------------
 
-#. In the :guilabel:`Filter` field, enter a filter document. You can use
-   all of the MongoDB :manual:`query operators
+#. In the :guilabel:`Filter` field, enter a filter document between the
+   curly braces. You can use all the MongoDB :manual:`query operators
```

Comment:
[nit] looks like the word "of" got left out from the original copy
```suggestion
   curly braces. You can use all of the MongoDB :manual:`query operators
```

Reply:
I took "of" out for minimalism (on vale's suggestion to pass the check)